### PR TITLE
Mgred/rapid.js 1

### PR DESCRIPTION
**Pull request by mgred**

### CODE REVIEW

**Comment by mgred** [comment]
#29

File: src/core/core.js [hunk 1]
```diff
@@ -140,6 +157,10 @@ class Core {
     return this;
   }
 
+  get interceptors () {
```
Comment (drewjbartlett) [inline src/core/core.js]: My only comment here is why do we need a getter for this?

**Comment by drewjbartlett** [comment]
@mgred I will need to document this in the docs. I am also rewriting this in all in typescript for v2 so this will have to be updated there as well :D. If you are familiar with TS I would love any help on the rewrite :)

File: src/core/core.js (hunk 1 above)
Comment (mgred) [reply src/core/core.js]: Just put this there to be access them easier e.g.:
```javascript
boot(){
    this.interceptors.request.push(...);
}
```

**Comment by mgred** [comment]
@drewjbartlett Yes, I'am quite familiar with TS and also willing to help you with any work to do :+1:

**Comment by drewjbartlett** [comment]
@mgred awesome! You can have a look at this branch: https://github.com/rapidjs/rapid.js/tree/v2-refactor

It is very very much a WIP. I do have some hiccups with properly making type definitions and such. That might be where I could use some help. I'll try to get back on this soon so maybe we could collaborate 😄

**Comment by mgred** [comment]
@drewjbartlett Sounds good! I will check out the branch and have a look. Thanks for the invite.

**Pull request merged**